src/chain.h

```diff
@@ -243,7 +243,7 @@ class CBlockIndex
 
     //! Public coin values of mints in this block, ordered by serialized value of public coin
     //! Maps <denomination,id> to vector of public coins
-    std::map<std::pair<sigma::CoinDenomination, int>, std::vector<sigma::PublicCoin>> sigmaMintedPubCoins;
+    std::map<std::pair<std::uint8_t, int>, std::vector<sigma::PublicCoin>> sigmaMintedPubCoins;
```

Comment:
I think it'd be much safer to just leave `sigma::CoinDenomination` as is to be safe. It's not clear if `std::uint8_t` is always a good replacement